<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vec4Input</class>
 <widget class="QWidget" name="Vec4Input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Field Name</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="mainSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelX">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(255, 62, 65);
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputX">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerXY">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelY">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(41, 211, 41);
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputY">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerYZ">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelZ">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(49, 38, 206);
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputZ">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerZW">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(72, 72, 72);
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>W</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
